### PR TITLE
fix(workflow): patch release workflow

### PR DESCRIPTION
**Pull request by CharlieS1103**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -100,15 +100,15 @@ jobs:
       - name: Extract version
         id: extract-version
         run: |
-          echo "tag_name=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "tag-name=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
```
Comment (theRealPadster) [inline .github/workflows/build.yml:103]:
```suggestion
          echo "tag_name=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -100,15 +100,15 @@ jobs:
...
       - name: Update Homebrew tap
         uses: mislav/bump-homebrew-formula-action@v2
         if: "!contains(github.ref, '-')"
         with:
           formula-name: spicetify-cli
           formula-path: spicetify-cli.rb
           homebrew-tap: spicetify/homebrew-tap
-          tag-name: ${{ env.tag_name }}
+          tag-name: ${{ env.tag-name }}
```
Comment (theRealPadster) [inline .github/workflows/build.yml:111]:
```suggestion
          tag-name: ${{ env.tag_name }}
```

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -100,15 +100,15 @@ jobs:
...
           base-branch: main
           download-url: https://github.com/spicetify/spicetify-cli/archive/${{ steps.extract-version.outputs.tag-name }}.tar.gz
```
Comment (theRealPadster) [inline .github/workflows/build.yml:113]:
```suggestion
          download-url: https://github.com/spicetify/spicetify-cli/archive/${{ steps.extract-version.outputs.tag_name }}.tar.gz
```

**Review by theRealPadster — CHANGES_REQUESTED** [review]
I think this is what we want?

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -110,7 +110,7 @@ jobs:
           homebrew-tap: spicetify/homebrew-tap
           tag-name: ${{ env.tag_name }}
           base-branch: main
-          download-url: https://github.com/spicetify/spicetify-cli/archive/${{ steps.extract-version.outputs.tag-name }}.tar.gz
+          download-url: https://github.com/spicetify/spicetify-cli/archive/${{ steps.extract-version.outputs.tag_name }}.tar.gz
```
Comment (ririxidev) [inline .github/workflows/build.yml:113]:
```suggestion
          download-url: https://github.com/spicetify/spicetify-cli/archive/${{ env.tag_name }}.tar.gz
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

**Review by kyrie25 — APPROVED** [review]

**Pull request merged**